processing/src/main/java/io/druid/segment/ColumnSelector.java

```diff
@@ -31,5 +31,5 @@
   List<String> getColumnNames();
 
   @Nullable
-  Column getColumn(String columnName);
+  ColumnHolder getColumn(String columnName);
```

Comment:
Nit, maybe this should be called `getColumnHolder` or something to be a bit clearer since `ColumnHolder.getColumn` also exists and there are a handful of places that are effectively calling `selector.getColumn().getColumn()` which reads kind of funny.

Reply:
Thanks for noticing, renamed.